packages/lodestar/src/chain/blocks/verifyBlock.ts

```diff
@@ -204,6 +204,21 @@ export async function verifyBlockStateTransition(
           chain.forkChoice.validateLatestHash(execResult.latestValidHash, null);
         }
         break;
+      case ExecutePayloadStatus.ELERROR:
+        // There can be many reasons for which EL failed some of the observed ones are
+        // 1. Connection refused / can't connect to EL port
```

Comment:
As discussed in EthR&D network errors must not be treated the same as EL being unable to process the block